phys2bids/tests/test_integration.py

```diff
@@ -2,12 +2,13 @@
 import json
 import math
 import os
-from pkg_resources import resource_filename
 import re
 import subprocess
 
```

Comment:
```suggestion
```

`pkg_resources` _is_ part of the standard library and should be bundled with the standard imports. It mostly is here, but we just need to get rid of this extra blank line!

Reply:
This also still stands! `pkg_resources` shouldn't be separated from standard library imports.